app/js/components/manageApps/ManageApps.jsx

```diff
@@ -619,7 +619,6 @@ export default class ManageApps extends React.Component {
     if (addon && addon.appDetails.type === 'OWA') {
       this.requestUrl = '/owa/installapp';
       let urlObject = {
-        'fileName': addon.appDetails.name,
```

Comment:
What is the reason behind this change? The ticket description is not clear.

Reply:
The owa installation api call has been updated to take only urlValue. So this ticket is to update the post data sent during the call to the api